### ISSUE TITLE
Pika is not compatible with python 3.7

### ISSUE THREAD

**Issue by aolefira** [pika/pika#921]
"async and await are not recommended to be used as variable, class, function or module names. Introduced by PEP 492 in Python 3.5, they will become proper keywords in Python 3.7." https://docs.python.org/3.7/whatsnew/3.5.html#new-keywords

So, you catch error in python 3.7:
/envs/plarin-3.7.0a4/lib/python3.7/site-packages/pika/adapters/libev_connection.py", line 106
    self.async = None
             ^
SyntaxError: invalid syntax

**Comment by lukebakken** [comment]
We would be glad to accept a PR to address this, thanks!

**Comment by clarabez** [comment]
Had the same using python 3.9. Just downgraded to 3.6 (into my venv) and worked fine.

**Comment by lukebakken** [comment]
@clarabez could you provide more detail? Specifically, what errors? I just ran the test suite successfully using Python 3.9.1 -

```
---------------------------------------------------------------------------------
TOTAL                                             6888   2394   1506    164    66%
----------------------------------------------------------------------
Ran 1181 tests in 61.305s

OK (SKIP=61)
(venv) lbakken@shostakovich ~/development/pika/pika (lrb-update-python-versions=)
$ python --version
Python 3.9.1
```

**Comment by meaditisharma** [comment]
Hi, 

I am also facing this issue. 

`Python 3.9.9 (main, Nov 21 2021, 03:23:42) 
[Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from broker_rabbit import BrokerRabbitMQ
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/saditi/workspace/nstriagetodb/broker_rabbit/__init__.py", line 1, in <module>
    from broker_rabbit.broker import BrokerRabbitMQ
  File "/Users/saditi/workspace/nstriagetodb/broker_rabbit/broker.py", line 4, in <module>
    from broker_rabbit.channels import ProducerChannel
  File "/Users/saditi/workspace/nstriagetodb/broker_rabbit/channels.py", line 4, in <module>
    import pika
  File "/Users/saditi/workspace/nstriagetodb/venv/lib/python3.9/site-packages/pika/__init__.py", line 22, in <module>
    from pika.adapters import BaseConnection
  File "/Users/saditi/workspace/nstriagetodb/venv/lib/python3.9/site-packages/pika/adapters/__init__.py", line 39, in <module>
    from pika.adapters.libev_connection import LibevConnection
  File "/Users/saditi/workspace/nstriagetodb/venv/lib/python3.9/site-packages/pika/adapters/libev_connection.py", line 103
    self.async = None
         ^
SyntaxError: invalid syntax
>>> 
`

**Comment by lukebakken** [comment]
@meaditisharma this was fixed in #924. Please upgrade Pika.

**Comment by meaditisharma** [comment]
@lukebakken Thanks, It got fixed with pika update
